Add support for Symfony 6 & 7

zero2one opened the pull request:
Add support for:

- PHP 8.3
- Symfony 6.4.x
- Symfony 7.x
- PHPUnit 9 & 10.

Drop support for:

- Symfony 6.3 and lower.
- PHP 8.2 and lower.

Updated:

- Directory structure.
- Unit tests.
- Code impacted by Symfony updates.

zero2one committed:
Update to PHP 8.3 and Symfony6+

zero2one committed:
Refactor the bundle directory structure

zero2one committed:
Update Uid extractor for Symfony6 and beyond

Adds also support for other record ID methods.

zero2one committed:
Update tests for Symfony6+

zero2one committed:
Update Travis-CI for PHP 8.3

Jelle-S reviewed (approved):
Reluctantly, because I still hate final classes and private properties... But, sure.

zero2one committed:
Update code for Symfony 6 & 7

- Use environment variables.
- Use LogRecord instead of arrays to process log records.
- Add config (services & parameters) from within Bundle file.
- Add Monolog syslog handler config from within Bundle file.
- Removed obsolete files.

zero2one committed:
Fix error about not defined parameter

zero2one committed:
Testing out proper arguments string
